test/langtools/tools/javap/malformed/MalformedTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
isn't this test testing the same as: `test/langtools/tools/javap/8260403/T8260403.java`?